src/tokens/ERC20Vault.sol

```diff
@@ -2,16 +2,15 @@
 pragma solidity >=0.8.0;
 
 import {ERC20WithSupply} from "BoringSolidity/ERC20.sol";
-import "BoringSolidity/BoringOwnable.sol";
 import "BoringSolidity/libraries/BoringERC20.sol";
 import "interfaces/IERC20Vault.sol";
 
 /// @notice A token that behaves like SushiBar where the contract underlying token balance
 /// influences the share value.
-contract ERC20Vault is IERC20Vault, ERC20WithSupply, BoringOwnable {
+contract ERC20Vault is IERC20Vault, ERC20WithSupply {
     using BoringERC20 for IERC20;
 
-    IERC20 public immutable underlying;
+    IERC20 public underlying;
```

Comment:
why was this change necessary if not used for wrapper?

Reply:
If GMX changes the stakedGlp for an updated version or bug fixed (like they did in the past), we can change it here without deprecating the wrapper and the cauldron

Reply:
good catch, forgot to put it back to immutable.